src/pages/Pokemon/pokemon.css

```diff
@@ -1,102 +1,59 @@
+body {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+  align-items: center;
+  background-image: white;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  color: 'black';
+  font-family: 'Exo', sans-serif;
+  background:#fff url("https://assets.pokemon.com/static2/_ui/img/chrome/container_bg.png")
+}
 
+.container {
+  background-color: white;
+  /* display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr; */
+  padding: 16px;
+  color: 'black';
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+}
 
-#root {
-    width: 100%;
-    height: 100%;
+
+
+h1 {
+  color: #919191;
+  cursor: default;
+  font-size: 187.5%;
+  line-height: 125%;
+  font-weight: 100;
+  padding: 32px;
 }
-.pokemonPage {
+
+/* .pokemon-page {
   display: grid;
   grid-template-columns: repeat(3, 1fr);
-}
+} */
 
-.pokemonId {
+.pokemon-id {
     color: #919191;
     font-weight: 400;
     font-size: 36px;
     
   }
   
-  .pokemonName {
+  .pokemon-name {
```

Comment:
mate I think the exact same style has been copied below, and cos it's. lower down in the file it will override these styles mate